assets/js/src/core/modules/app/theme/utils/themes/default-theme.ts

```diff
@@ -182,6 +183,8 @@ const defaultTheme = {
       lineWidth: 1,
       marginXS: 8,
       paddingXXS: 4,
+      colGapXXS: 4,
+      rowGapXXS: 4,
```

Comment:
Do we still need them? I think they are not in use? Also can't see them in Theme buddy

Reply:
I've deleted it, thanks.